### DISCUSSION TITLE
The typedRoutes types suggestions from external libs folder

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [nrwl/nx#19719] [General]
Is there any way to make it work on NX monorepo that will gives the Next.js 13 typedRoutes types suggestions to the external libs folders?

**Comment by jaysoo** [comment]
You can, but you have to include the types in the library's tsconfig file.

e.g.

```json5
{
  //...
  "include": [
    "src/**/*.js",
    "src/**/*.jsx",
    "src/**/*.ts",
    "src/**/*.tsx",
    "../../apps/demo/.next/types/**/*.ts",
    "../../dist/apps/demo/.next/types/**/*.ts"
  ]
}
```

The path is relative to the library's tsconfig file, so in this case I have to use `../../`.

**Reply by MuttakinHasib** [reply]
Thanks @jaysoo ❤️
